week2/2-website/css/style.css

```diff
@@ -1,19 +1,261 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+*{
 
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+}
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    height: 100vh;
+}
+
+
+.devices{
+    
+    display:flex;
+    width: 60%;
+    margin:auto;
+    justify-content: space-around;
+
 }
+.devices li{
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+    text-align: center;
+}
+header {
+width: 100%;
+background-color: white;
+height: 6vh;
+}
+.content-nav {
+    align-items: center;
+    height: 100%;
 
+}
+.links {
+ width: 80% ;
+ justify-content: flex-end;
+ height: 100%;
+ align-items: center;
+}
+.rapper {
+    border:1px solid green;
+    display: flex;
+    width: 70%;
+    margin:auto;
+    justify-content: space-between;
+    align-items: center;
+}
+nav{
+    position: fixed;
+    width: 100%;
+    border-bottom: 0.001rem solid black;
+    display: flex;
+    text-decoration: none;
+    border: 1px solid red;
+    justify-content: space-around;
+    }
+    nav-content {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    align-items: centerjustify;
+    padding: 0.5rem;
+    height: 5rem;
+    margin: 0.5rem;
+    
+    }
+    nav-list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style-type: none;
+    font-size: 1.1rem;
+    padding: 0.5rem;
+    margin:0.3rem;
+    list-style : none;
+}
+    nav a {
+        text-decoration: none;
+        color: rgb(226, 213, 213);
+    
+    }
+
+    .links {
+        list-style:none;
+        display: flex;
+        flex-direction: row;
+        align-items: flex-end;
+        
+    }
+
+.nav-link {
+
+    padding: 3rem;
+}
+    #logo {
+    height: 2.75rem;
+    width: 2.75rem;
+    margin-right: 9rem;
+    }
+    .introduction {
+        background: url(../img/first-background.jpg)no-repeat center;
+        padding: 25%;
```

Comment:
There's a lot of padding here to control the size of this background image.

Instead you can use a height variable such as `height: 60vh` or `min-height: 600px`.